### ISSUE TITLE
Invalid entry for `@angular/core/testing`

### ISSUE THREAD

**Issue by Quramy** [fuse-box/fuse-box#484]
Hello. I've tried bundle Angular2 app unit test. But the created bundle does not work...

I think there are some issues about bundling `@angular/core/testing` package.

This issue can be reproduced with the following 2 files(reproducing repo is [here](https://github.com/Quramy/angular-testing-require)):

- src/main.ts:

```ts
/* src/main.ts */
import { getTestBed } from "@angular/core/testing";

console.log(getTestBed());
```

- fuse.js

```js
const fsbx = require("fuse-box");

const fuse = fsbx.FuseBox.init({
  homeDir: "src",
});

fuse.bundle({
  "dist/bundle.js": "> main.ts",
});
```

Fuse-box outputs the following:

```js
(function(FuseBox){FuseBox.$fuse$=FuseBox;
FuseBox.pkg("default", {}, function(___scope___){
___scope___.file("main.js", function(exports, require, module, __filename, __dirname){

"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var testing_1 = require("@angular/core/testing");
console.log(testing_1.getTestBed());

});
});
FuseBox.pkg("@angular/core", {}, function(___scope___){
return ___scope___.entry = "bundles/core.umd.js";
});

FuseBox.import("default/main.js");
FuseBox.main("default/main.js");
})
:
```

And executing the above bundle, an error,  `TypeError: Cannot read property 'getTestBed' of null`, occurs.

I think the line `FuseBox.pkg("@angular/core", {}, function(___scope___){` is invalid. Because `@angular/core/testing` has the following structure:

```txt
node_modules/@angular/core/testing
├── index.d.ts
├── index.metadata.json
├── package.json
└── src

1 directory, 22 files
```

and the package.json is:

```json
{
  "name": "@angular/core/testing",
  "typings": "../testing.d.ts",
  "main": "../bundles/core-testing.umd.js",
  "module": "../@angular/core/testing.es5.js",
  "es2015": "../@angular/core/testing.js"
}
```

The package.json's  `main` points a entry file `../bundles/core-testing.umd.js`. 

I want fuse-box to output such as: 

```js
:
FuseBox.pkg("@angular/core/testing", {}, function(___scope___){
return ___scope___.entry = "../bundles/core-testing.umd.js";
});
:
```

How do I configure bundle's entry point about specific packages? Or is there some workadounds?

**Comment by nchanged** [comment]
hi!
have you tried aliases?
```
alias : {
  “@angular/core/testing” : “@angular/core/bundles/core-testing.umd.js"
}
```

Something like that

**Comment by Quramy** [comment]
Thanks @nchanged .

Great, it works well for me!

**Comment by nchanged** [comment]
Fixed without aliases in `2.1.0-beta.4`
